README.md

```diff
@@ -17,7 +17,7 @@ Mistral.rs is a fast LLM inference platform supporting inference on a variety of
 - More models: please submit requests [here](https://github.com/EricLBuehler/mistral.rs/issues/156).
 - X-LoRA: Scalings `topk` and softmax `topk` ([#48](https://github.com/EricLBuehler/mistral.rs/issues/48)).
 - Parallel linear layers (sharding) ([#50](https://github.com/EricLBuehler/mistral.rs/issues/50)).
-- Speculative decoding: https://arxiv.org/pdf/2211.17192
+- Vision models: Idefics 2 ([#309](https://github.com/EricLBuehler/mistral.rs/pull/309)).
```

Comment:
Was it intentional to drop the Speculative decoding line?

I don't have much context for it's previous inclusion beyond an earlier direct commit from you @EricLBuehler 

https://github.com/EricLBuehler/mistral.rs/commit/b3d130dcf7432bbbe4119d3d6ad269b4d3214e29

Reply:
@polarathene, yes, we have speculative decoding: https://github.com/EricLBuehler/mistral.rs/blob/master/examples/python/speculative.py